internal/services/rewards.go

```diff
@@ -156,15 +157,17 @@ func (t *BaselineClient) assignPoints() error {
 		return err
 	}
 
-	lastWeekByDevice := make(map[string]*models.Reward)
+	lastWeekByDevice := make(map[int64]*models.Reward)
 	for _, reward := range lastWeekRewards {
```

Comment:
There is a slight problem here in that `reward.UserDeviceTokenId` might be null and I'm not sure what'll happen. I think it will not be good.

Let me just get rid of all these weird types. Any device that still isn't minted at this point has not earned for a long time and will never earn. It's not worth carrying around in the DB.